library/src/webapp/js/lang-datepicker/lang-datepicker.js

```diff
@@ -1850,9 +1876,36 @@ $.datepicker._gotoToday = function (id) {
 			// set the initial date for the picker
 			initDateTime();
 
-			// instantiate datetimepicker
+			// Instantiate datetimepicker
 			localDTPicker = $(options.input).css('min-width','200px').datetimepicker(cfg);
 
+			var openDatepickerLabel = $.datepicker._defaults['openDatepicker'] || 'Open Datepicker';
+			$(options.input).next('.ui-datepicker-trigger').attr('aria-label', openDatepickerLabel);
+
+			var ariaLiveDiv = $('<div>', {
+				id: 'datepicker-aria-live',
+				'aria-live': 'polite',
+				'class': 'visually-hidden'
+			}).appendTo('body');
+
+			var datePickerTrigger = $(options.input).next('.ui-datepicker-trigger');
+			datePickerTrigger.attr('aria-haspopup', 'true').attr('aria-expanded', 'false');
```

Comment:
I think you can just chain this straight off

Reply:
Okay. Thanks. I will do that.